test/refunder-test.ts

```diff
@@ -0,0 +1,295 @@
+
+import { defaultAccounts } from "@ethereum-waffle/provider";
+import { BigNumber } from "@ethersproject/bignumber";
+import { Contract } from "@ethersproject/contracts";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { expect } from "chai";
+import { ethers } from "hardhat";
+
+import { 
+	INSUFFICIENT_BALANCE, 
+	NOT_AN_OWNER,
+	NOT_REFUNDABLE
+} from './constants/error-messages.json';
+
+const ethToWei = (ether: string) => {
+	return ethers.utils.parseEther(ether);
+}
+
+describe("Refunder", function() {
```

Comment:
I think that we should assert that certain types of events are emitted on transactions